tests/accessor/accessor_default_values.h

```diff
@@ -0,0 +1,253 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This file provides functions for tests on accessor default values.
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+#define SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+
+#include "accessor_common.h"
+
+#include <type_traits>
+
+namespace accessor_default_values_test {
+
+namespace detail {
+
+constexpr int expected_dims = 1;
+constexpr auto expected_target = sycl::target::device;
+constexpr auto expected_placeholder = sycl::access::placeholder::false_t;
+template <typename T>
+constexpr auto expected_mode =
+    std::is_const_v<T> ? sycl::access_mode::read : sycl::access_mode::read_write
+
+}  // namespace detail
+
+/**
+ * @brief Provides verification that local_accessor's dimension default value
+ *        is equal to expected one
+ * @tparam T Current data type
+ * @param type_name Current data type string representation
+ */
+template <typename T>
+void verify_local_acc_dim_value(const std::string& type_name) {
+  auto section_name = get_section_name<detail::expected_dims>(
+      type_name,
+      "Verify default value for dimensions "
+      "template parameter. [local_accessor]");
+  SECTION(section_name) {
+    REQUIRE(std::is_same_v<sycl::local_accessor<T>,
+                           sycl::local_accessor<T, detail::expected_dims>>);
+  }
+}
+
+/**
+ * @brief Provides verification that local_accessor's dimension default value
+ *        is equal to expected one
+ * @tparam T Current data type
+ * @param type_name Current data type string representation
+ */
+template <typename T>
+void verify_host_acc_dim_value(const std::string& type_name) {
+  auto section_name = get_section_name<detail::expected_dims>(
+      type_name,
+      "Verify default value for dimensions "
+      "template parameter. [local_accessor]");
+  SECTION(section_name) {
+    REQUIRE(std::is_same_v<sycl::local_accessor<T>,
+                           sycl::local_accessor<T, detail::expected_dims>>);
+  }
+}
```

Comment:
Shouldn't this function test host accessor? If so, there are a lot of copy-paste errors where `local` has to be replaced with `host`.

Reply:
> Shouldn't this function test host accessor? 

Yes, replaced with `sycl::host_accessor` [here](https://github.com/KhronosGroup/SYCL-CTS/pull/335/commits/20bb630b11d3e25acee89b37b3aacde18ae3fda1#diff-9ed3eae8f2889bc04dfc0fad247a8fa4e381e1fff4144e0f1a23670d108f6fe9R60-R61)

> If so, there are a lot of copy-paste errors where local has to be replaced with host.

Where? I think that in all other cases all is good

Reply:
> > If so, there are a lot of copy-paste errors where local has to be replaced with host.
> 
> Where? I think that in all other cases all is good

In the part I selected for this comment.